aarch64-unknown-none.ld

```diff
@@ -0,0 +1,44 @@
+ENTRY(ram64_start)
+
+/* Cloud Hypervisor Memory layout:
+    DRAM:   [0x4000_0000-0xfc00_0000]
+    FDT:    [0x4000_0000-0x401f_ffff)
+    ACPI:   [0x4020_0000-0x403f_ffff)
+    kernel: [0x4048_0000-]
+   The stack start is at the end of the DRAM region. */
+ram_min = 0x40480000;
+
+SECTIONS
+{
+  /* Mapping the program headers and note into RAM makes the file smaller. */
+  . = ram_min;
+
+  /* These sections are mapped into RAM from the file. Omitting :ram from
+     later sections avoids emitting empty sections in the final binary.       */
+  code_start = .;
+  .text.boot  : { *(.text.boot)        }
+  .text       : { *(.text .text.*)     }
+  . = ALIGN(4K);
```

Comment:
```suggestion
  . = ALIGN(64K);
```

Paging and TTable code assumes 64KiB pages, but you are aligning to 4KiB here (rest of file as well)